CONTRIBUTING.md

```diff
@@ -15,13 +31,19 @@ See [details on our policy on Code of Conduct](./doc/guides/contributing/code-of
 
 ## [Issues](./doc/guides/contributing/issues.md)
 
+Issues in `nodejs/node` are the primary means by which bug reports and
```

Comment:
I don't feel strong about this, but I think having a short description is better

Reply:
I agree having a description is better.
Maybe if it's worth mentioning https://github.com/nodejs/node/discussions?

Reply:
I agree having a description is better.
Maybe if it's worth mentioning https://github.com/nodejs/node/discussions?

Reply:
https://github.com/nodejs/node/discussions/38057

Reply:
I meant adding a section `## Discussions` that introduces what are discussions and how to contribute on them. It can also be done in a follow up PR.

Reply:
Maybe we can add a section with this discussion's content https://github.com/nodejs/node/discussions/35662

and give a link to discussions page